src/nnet3/nnet-utils.cc

```diff
@@ -745,23 +1064,22 @@ class SvdApplier {
     component_b->SetUpdatableConfigs(affine);
     *component_a_out = component_a;
     *component_b_out = component_b;
+    return true;
   }
 
   // This function modifies the topology of the neural network, splitting
   // up the components we're modifying into two parts.
   // Suppose we have something like:
   //  component-node name=some_node component=some_component input=
+  // nodes_to_split will be a list of component-node indexes that we
```

Comment:
I think the previous commenting way right next to the variables is better. Also use the right name nodes_to_modify instead of nodes_to_split.